src/pages/DogEdit.js

```diff
@@ -56,17 +56,20 @@ const DogEdit = ({ dogs, updateDog }) => {
                 />
             </FormGroup>
             <FormGroup>
-               <Label for="image">Image URL</Label>
-               <Input 
-               type="text" 
-               id="image"
-               name="image"
-               onChange={handleChange}
+                <Label for="image">Image URL</Label>
+                <Input 
+                type="text" 
+                id="image"
+                name="image"
+                onChange={handleChange}
```

Comment:
Make sure to look over indentation for attributes.

Reply:
I'm using prettier and that's how its indenting the code .